src/components/tooltip/tooltip.js

```diff
@@ -102,12 +103,20 @@ function MdTooltipDirective($timeout, $window, $$rAF, $document, $interpolate,
       }
     }
 
-    function addAriaLabel(override) {
-      if (override || !parent.attr('aria-label')) {
-        // Only interpolate the text from the HTML element because otherwise the custom text
-        // could be interpolated twice and cause XSS violations.
-        var interpolatedText = override || $interpolate(element.text().trim())(parent.scope);
+    function addAriaLabel(labelText) {
+      // Only interpolate the text from the HTML element because otherwise the custom text could
+      // be interpolated twice and cause XSS violations.
+      var interpolatedText = labelText || $interpolate(element.text().trim())(parent.scope);
+
+      // Only add the `aria-label` to the parent if there isn't already one, or if the previous
+      // `aria-label` was added by the tooltip directive.
+      if (!parent.attr('aria-label') || parent.attr('aria-labelledby') === tooltipId) {
         parent.attr('aria-label', interpolatedText);
```

Comment:
If we are going to use `aria-labelledby`, we don't need to use `aria-label`. Therefore, we don't have to interpolate the text at all.